Src/UUIDNext/Generator/UuidNameGeneratorBase.cs

```diff
@@ -22,15 +40,15 @@ public Guid New(Guid namespaceId, string name)
 
             //Compute the hash of the name space ID concatenated with the name.
             int bytesToHashCount = namespaceBytes.Length + utf8NameBytes.Length;
+
```

Comment:
I'll revert these new line changes so that it would look as if I only added the netframework block above